modules/storage/storage-accounts/local-users/version.json

```diff
@@ -1,8 +1,7 @@
 {
-  "$schema": "https://aka.ms/bicep-registry-module-version-file-schema#",
-  "version": "0.1",
-  "pathFilters": [
-    "./main.json",
-    "./metadata.json"
-  ]
+    "$schema": "https://aka.ms/bicep-registry-module-version-file-schema#",
+    "version": "0.4",
```

Comment:
I see version bumped if previously at `0.1`. Not against that just pointing that out in case it wasn't intended.

Reply:
I think that is just a result of the search and replace. Personally I don't care, but if you feel it's worth it, I can tryi to recover the originals :)

Reply:
As said, just wanted to raise it, not worthy the effort of recovering. As mentioned many times, this is not the CARML module official version, but a feature of the CI environment.